client/app/scripts/superdesk/superdesk.js

```diff
@@ -97,8 +97,13 @@ define([
         }]);
 
     // make sure there is templates module defined
-    angular.module('templates', []);
-    modules.push('templates');
+    try {
```

Comment:
actually, its templates cache for all superdesk modules.  probably does make more sense to move the try to main.js instead of superdesk.js, I'll make the change.